src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/CommentHelper.java

```diff
@@ -162,7 +164,10 @@ Element getElement(ReferenceTree rtree) {
             return null;
         }
         DocTrees doctrees = configuration.docEnv.getDocTrees();
-        return doctrees.getElement(docTreePath);
+        // Workaround JDK-8284193
```

Comment:
Nit: either split "workaround" or add a preposition ("for"?) before the issue type. It's either a phrasal verb or a noun with a preposition.

Otherwise that phrase reads weird.

Reply:
done